docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -696,9 +697,123 @@ To read and write a value of the Bluetooth descriptor:
 
    descriptor.writeValue(newValue, writeSuccess, writeFail);
    ```
+<!-- TODO #04
+      - write about getAttMtu()
+      - write about requestAttMtuChange()
+ -->
+### Accessing the ATT MTU of connected device
+<!-- END #04 -->
+
+<!-- TODO #05
+      - write about addAttMtuChangeListener()
+      - write about removeAttMtuChangeListener()
+ -->
+### Receiving Notifications on ATT MTU changes
+<!-- END #05 -->
+
+
+## Managing the local GATT server
+
+<!-- TODO #06
+      - write about start()
+ -->
+### Start the server
+<!-- END #06 -->
+
+<!-- TODO #07
+      - write about stop()
+ -->
+### Stop the server
+<!-- END #07 -->
+
+<!-- TODO #08
+      - write about addConnectStateChangeListener()
+      - write about removeConnectStateChangeListener()
+ -->
+### Receiving notifications on server's Connection State changes
+<!-- END #08 -->
+
+<!-- TODO #09
+      - write about getConnectionMtu()
+ -->
+### Accessing connection's ATT MTU
+<!-- END #09 -->
+
+<!-- TODO #10
+      - write about registerService()
+ -->
+### Registering services
+<!-- END #10 -->
+
+<!-- TODO #11
+      - write about unregisterAllServices()
+      - write about BluetoothGATTServerService::unregister()
+ -->
+### Unregistering services
+<!-- END #11 -->
+
+<!-- TODO #12
+      - write about setReadValueRequestCallback() for characteristic
+      - write about setWriteValueRequestCallback() for characteristic
+ -->
+### Managing characteristics of server service
```

Comment:
Consider merging the sections about read/write value request callbacks for characteristics and descriptors - the callbacks are set in the same way for both types of GATT entities. Also, I think, we can give this section a name that describes it more precisely, e.g. "Responding to read and write value requests from server clients"